src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
```

Comment:
This is probably worth discussing.  I think the goal should be that we are not required to manually update the lab machine as this script changes.  If we have a simple "bootstrap" script that only knows how to 'git clean' and 'git pull' and then executes a script in our repo to launch the builds, we will have a self-maintaining machine.  Think through the scenario of changing the contents of this CMD file -- what are the steps you would need today to update the lab machine to use it? With my proposal, the 'git pull' will get the new copy by default.